README.md

```diff
@@ -13,19 +13,25 @@ Despite being written from scratch, this project was inspired by the (now archiv
 - Never tested on Windows (but should work).
 - Does not generate EOT font (unecessary, woff and woff2 cover all modern browsers as well as IE9 and IE10).
 - There is no CLI, only a Node.JS API (PR welcome).
-- There is no way to pass down options to the tools used under the hood (`svgicons2svgfont`, `svg2ttf`, `ttf2woff` and `wawoff2`).
+- There is no way to pass down options to all of the tools used under the hood (`svg2ttf`, `ttf2woff` and `wawoff2`).
```

Comment:
"to **some**" and not "to **all**" since we can pass down options to `svgicons2svgfont`, right?